### PR TITLE
Refactoring/rename constants and enum types

### PR DESCRIPTION
**Pull request by CiprianDraghici**

### CODE REVIEW

File: src/UI/Denominate/index.tsx [hunk 1]
```diff
@@ -1,11 +1,7 @@
 import React from 'react';
 import classNames from 'classnames';
 import globalStyles from 'assets/sass/main.scss';
-import {
-  decimals as configDecimals,
-  denomination as configDenomination,
-  mainnetEgldLabel
-} from 'constants/index';
+import { DECIMALS, DENOMINATION, mainnetEgldLabel } from 'constants/index';
```
Comment (arhtudormorar) [inline src/UI/Denominate/index.tsx:4]: if we make an exception for  `mainnetEgldLabel`, we should also keep `decimals` and `denomination`

File: src/UI/TransactionsTable/TransactionReceiver.tsx [hunk 2]
```diff
@@ -20,7 +20,7 @@ export const TransactionReceiver: React.FC<TransactionReceiverProps> = ({
   showLockedAccounts
 }) => {
   const directionIn =
-    transaction.transactionDetails.direction === TransactionDirection.IN;
+    transaction.transactionDetails.direction === TransactionDirectionEnum.IN;
```
Comment (arhtudormorar) [inline src/UI/TransactionsTable/TransactionReceiver.tsx:23]: rename `TransactionDirectionEnum` to `TxDirectionEnum`

File: src/UI/TransactionsTable/TransactionSender.tsx [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
 import React from 'react';
+import classNames from 'classnames';
 import {
   ExtendedTransactionType,
-  TransactionDirection
+  TransactionDirectionEnum
 } from 'components/TransactionsInterpreter/helpers/types';
-import { ShardSpan } from './ShardSpan';
+import globalStyles from '../../assets/sass/main.scss';
+import { addressIsValid } from '../../utils';
```
Comment (arhtudormorar) [inline src/UI/TransactionsTable/TransactionSender.tsx:8]: use absolute import

File: src/components/TransactionsInterpreter/helpers/getReceiptMessage.ts [hunk 4]
```diff
@@ -11,8 +11,8 @@ const getReceiptValue = (transaction: ServerTransactionType) => {
   if (transaction.receipt?.data === REFUNDED_GAS) {
     const denominatedGas = denominate({
       input: transaction.receipt.value,
-      denomination,
-      decimals,
+      denomination: DENOMINATION,
```
Comment (arhtudormorar) [inline src/components/TransactionsInterpreter/helpers/getReceiptMessage.ts:14]: keeping them small is an advantage for using them here

File: src/components/TransactionsInterpreter/helpers/tests/getOperationsMessages.test.ts [hunk 5]
```diff
@@ -17,7 +17,7 @@ describe('getOperationsMessages', () => {
     const baseTransactionOperation = {
       name: 'send',
       type: VisibleTransactionOperationType.egld,
-      action: TransactionOperationActionType.transfer,
+      action: TransactionOperationActionTypeEnum.transfer,
```
Comment (arhtudormorar) [inline src/components/TransactionsInterpreter/helpers/tests/getOperationsMessages.test.ts:20]: `TransactionOperationActionTypeEnum` -> `TxOperationActionTypeEnum`

File: src/components/TransactionsInterpreter/helpers/tests/getReceiptMessage.test.ts [hunk 6]
```diff
@@ -27,8 +27,8 @@ describe('getReceiptMessage', () => {
     const result = getReceiptMessage(transaction);
     const denominatedGas = denominate({
       input: transaction.receipt.value,
-      denomination,
-      decimals,
+      denomination: DENOMINATION,
```
Comment (arhtudormorar) [inline src/components/TransactionsInterpreter/helpers/tests/getReceiptMessage.test.ts:30]: I would say revert to small

File: src/components/TransactionsInterpreter/helpers/urlBuilder.ts [hunk 7]
```diff
@@ -1,26 +1,30 @@
...
   transactionDetailsLogs: (hash: string) => `/transactions/${hash}/logs`,
   nodeDetails: (publicKey: string) => `/nodes/${publicKey}`,
   accountDetails: (address: string) => `/accounts/${address}`,
   accountDetailsContractCode: (address: string) => `/accounts/${address}/code`,
   accountDetailsTokens: (address: string) => `/accounts/${address}/tokens`,
   accountDetailsNfts: (address: string) => `/accounts/${address}/nfts`,
-  accountDetailsScResults: (address: string) => `/accounts/${address}/sc-results`,
-  accountDetailsContracts: (address: string) => `/accounts/${address}/contracts`,
+  accountDetailsScResults: (address: string) =>
+    `/accounts/${address}/sc-results`,
```
Comment (arhtudormorar) [inline src/components/TransactionsInterpreter/helpers/urlBuilder.ts:16]: use ENDPOINTS variables

File: src/components/TransactionsInterpreter/helpers/urlBuilder.ts [hunk 8]
```diff
@@ -1,26 +1,30 @@
...
+  accountDetailsContracts: (address: string) =>
+    `/accounts/${address}/contracts`,
   identityDetails: (id: string) => `/identities/${id}`,
   tokenDetails: (tokenId: string) => `/tokens/${tokenId}`,
   tokenDetailsAccounts: (tokenId: string) => `/tokens/${tokenId}/accounts`,
-  tokenDetailsLockedAccounts: (tokenId: string) => `/tokens/${tokenId}/locked-accounts`,
+  tokenDetailsLockedAccounts: (tokenId: string) =>
+    `/tokens/${tokenId}/locked-accounts`,
```
Comment (arhtudormorar) [inline src/components/TransactionsInterpreter/helpers/urlBuilder.ts:23]: TOKENS_ENDPOINT

File: src/components/TransactionsInterpreter/helpers/urlBuilder.ts [hunk 9]
```diff
@@ -1,26 +1,30 @@
...
   tokenDetailsRoles: (tokenId: string) => `/tokens/${tokenId}/roles`,
   collectionDetails: (identifier: string) => `/collections/${identifier}`,
   nftDetails: (identifier: string) => `/nfts/${identifier}`,
   providerDetails: (address: string) => `/providers/${address}`,
-  providerDetailsTransactions: (address: string) => `/providers/${address}/transactions`,
+  providerDetailsTransactions: (address: string) =>
+    `/providers/${address}/transactions`
```
Comment (arhtudormorar) [inline src/components/TransactionsInterpreter/helpers/urlBuilder.ts:29]: PROVIDERS_ENDPOINT

File: src/services/transactions/utils.ts [hunk 10]
```diff
@@ -1,15 +1,15 @@
...
 import { calculateFeeLimit } from 'utils/operations';
 
 export function calcTotalFee(transactions: Transaction[], minGasLimit: number) {
   let totalFee = new BigNumber(0);
 
   transactions.forEach((tx) => {
     const fee = calculateFeeLimit({
-      gasPerDataByte,
-      gasPriceModifier,
+      gasPerDataByte: GAS_PER_DATA_BYTE,
```
Comment (arhtudormorar) [inline src/services/transactions/utils.ts:11]: advantage in keeping small

File: src/UI/Denominate/index.tsx (hunk 1 above)
Comment (StanislavSava) [inline src/UI/Denominate/index.tsx:4]: We should probably also use upper case for mainnet label

File: src/UI/TransactionsTable/TransactionReceiver.tsx [hunk 11]
```diff
@@ -1,14 +1,14 @@
...
-import globalStyles from '../../assets/sass/main.scss';
-import { ExplorerLink } from '../ExplorerLink';
 import React from 'react';
+import classNames from 'classnames';
 import {
   ExtendedTransactionType,
-  TransactionDirection
+  TransactionDirectionEnum
 } from 'components/TransactionsInterpreter/helpers/types';
+import globalStyles from '../../assets/sass/main.scss';
```
Comment (StanislavSava) [inline src/UI/TransactionsTable/TransactionReceiver.tsx:7]: can we use absolute imports?

File: src/UI/Denominate/index.tsx (hunk 1 above)
Comment (CiprianDraghici) [reply src/UI/Denominate/index.tsx:4]: It's not an exception, it's a miss

**Pull request merged**